MainModule/Server/Core/Remote.lua

```diff
@@ -76,7 +76,6 @@ return function(Vargs, GetEnv)
 			DonorCapes = true;
 			ConsoleKeyCode = true;
 			SplitKey = true;
-			AntiClientIdle = true;
```

Comment:
Are you sure the client doesn't need this value? Have you double checked? It's not like this setting is super secret and nasty.